bandit/cli/__init__.py

```diff
@@ -0,0 +1 @@
+"""Bandit is a tool designed to find common security issues in Python code."""
```

Comment:
Even though not strictly documented in a coding conventions doc, we don't normally put docstrings in init.py files... unless there is code within it.

Reply:
Reverted